### ISSUE TITLE
Chord symbols can't be transposed separately from its whole measure

### ISSUE THREAD

**Issue by abariska** [musescore/MuseScore#17638]
### Issue type

Other type of issue

### Bug description

For now chord symbols can be transposed only when selecting whole measure with notes and chords and applying transposition 

### Steps to reproduce

1. Open the gp file 
[Layers, chords and meta.gp.zip](https://github.com/musescore/MuseScore/files/11511361/Layers.chords.and.meta.gp.zip)
2. Select a chord symbol
3. Set transposition  (e.g. to major second) and click Ok button
4. Pay attention to the symbol

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/101250347/5493c3e3-553b-486b-86f3-8ebf0f6ffa06

### MuseScore Version

MuseScore 4.1

### Regression

No.

### Operating system

macOS 13

**Comment by wizofaus** [comment]
I wanted this exact behaviour recently - to transpose just the chord symbols and not the notes.  If it's agreed just having chord symbols selected and applying a transpose should cause this behaviour I'm happy to look into it.  As @Jojo-Schmitz mentioned it's not GP-specific.

(I'd actually like to be able to even just use cursor keys to do chord symbol transpositions, but I can see that would be problematic because they're already needed to change the vertical positioning)

**Comment by cbjeukendrup** [comment]
> just having chord symbols selected and applying a transpose should cause this behaviour

This would make sense to me

**Comment by MarcSabatella** [comment]
Me too.  The only slightly awkward thing is managing the dialog - what should be grayed out, whether any wording needs tweaking.

**Comment by wizofaus** [comment]
Nope, it's simple, just need to change the dialog so you can access that option to transpose chord names depending on whether there *are* chord symbols present (in fact, it would be perfectly safe to simply always allow that option, but I figured it made sense to disable it if you've only selected notes).
One thing though: you can't select a *range* that ONLY includes chord symbols but not notes (even using the selection filter), so you can't use some of the options like transpose to a particular key.

**Comment by MarcSabatella** [comment]
I’m not saying it’s *hard*, just that it requires design work in the dialog, which means it’s not just a simple code change but has that additional complication.

**Comment by wizofaus** [comment]
Yes, it works fine. But unfortunately my suggestion of moving the notes into v2 in order to be able to select a range that doesn't include the notes *doesn't* work - it still transposes the notes too, despite them not being selected.  I don't know if that's worth fixing too, or reporting separately.

**Comment by abariska** [comment]
@wizofaus the solution is just a step towards to the main issue fix, so I'd not close the ticket

**Comment by wizofaus** [comment]
How so, what's the further fix required?

**Comment by abariska** [comment]
As described in the summary, the problem is it's not possible to transpose chord symbols separately from notes and wise versa. Seems it should be designed as another transpose option.
@oktophonie please, confirm or describe your plan

**Comment by wizofaus** [comment]
But that's exactly what I implemented! You can select just the chord symbols and transpose *just* those.

**Comment by abariska** [comment]
@wizofaus your fix works as expected technically but in context of the current Transpose dialog it looks as a hack with the same list of parameters as used for notes.
- 'Transposition to key' option is blocked
- 'Transpose chord symbols' option is present and still affects to the result (e.g. if I apply transposition to selected chord and the checkbox is unticked, there will be no changes). Which is weird.

Temporarily it can be like this but the dialog should be re-designed a bit. 
@bkunda please take a look, just fyi.

**Comment by wizofaus** [comment]
"Transposition to key" only works if you select a *range*, which has always been the case. "Transpose chord symbols" should be disabled if you have *only* selected chord symbols, if that's not the case then I agree, it should ideally be fixed, however this dialog is slated for replaced with a QML-based one, so at best that's a "nice-to-have".

**Comment by wizofaus** [comment]
Not likely to be looking at any MS stuff in the immediate future sorry @mikekirin
